backend/src/main/java/com/zpi/backend/user_opinion/UserOpinionController.java

```diff
@@ -44,18 +44,19 @@ public ResponseEntity<UserOpinion> addOpinion(Authentication authentication, @Re
 
     @Operation(
             summary = "Get opinions about user",
-            description = "Returns all opinions that were given to the user with a given UUID"
+            description = "Returns all opinions that were given to the user with a given UUID, everyone can use " +
+                    "this endpoint."
     )
 
     @GetMapping("/user/{uuid}/opinions")
-    public ResponseEntity<ResultsDTO<UserOpinion>> getOpinions(@PathVariable String uuid, @RequestParam int page, @RequestParam int size) throws UserDoesNotExistException {
+    public ResponseEntity<ResultsDTO<ReturnOpinionServiceDTO>> getOpinions(@PathVariable String uuid, @RequestParam int page, @RequestParam int size) throws UserDoesNotExistException {
```

Comment:
Please, change all method names and comments in sout from Opinion to UserOpinion (cause of GameInstanceOpinion)